src/Exizent.CaseManagement.Client/Models/EstateItems/FinancialGiftResourceRepresentation.cs

```diff
@@ -12,4 +12,5 @@ public class FinancialGiftResourceRepresentation : EstateItemResourceRepresentat
     public decimal ExemptionValue { get; init; }
     public DateTime? DateOfGift { get; init; }
     public FinancialGiftExemptionCategory? ExemptionCategory { get; init; }
```

Comment:
The FinancialGiftsDetails should not be on the FinancialGift itself.  You can imagine 10 gifts, but there will only ever be one FinancialGifts details per case..